modcc/parser.cpp

```diff
@@ -1304,13 +1306,13 @@ expression_ptr Parser::parse_conserve_expression() {
     return make_expression<ConserveExpression>(here, std::move(lhs), std::move(rhs));
 }
 
-expression_ptr Parser::parse_expression(int prec) {
+expression_ptr Parser::parse_expression(int prec, tok t) {
     auto lhs = parse_unaryop();
```

Comment:
A comment explaining what `t` means here, or a more explicit parameter name (e.g. `stop_token`?) would be a help to explain its function.

Reply:
I will rename it to `stop_token`